sdcm/scylla_bench_thread.py

```diff
@@ -145,14 +146,31 @@ def verify_results(self):
 
         return sb_summary, errors
 
-    def create_stress_cmd(self, stress_cmd):
+    def create_stress_cmd(self, stress_cmd, cmd_runner):
         if self.connection_bundle_file:
             stress_cmd = f'{stress_cmd.strip()} -cloud-config-path={self.target_connection_bundle_file}'
         else:
             # Select first seed node to send the scylla-bench cmds
             ips = ",".join([n.cql_address for n in self.node_list])
             stress_cmd = f'{stress_cmd.strip()} -nodes {ips}'
 
+            if self.params.get("client_encrypt"):
+                loader = self.loader_set.get_loader()
+                for ssl_file in loader.ssl_conf_dir.iterdir():
+                    if ssl_file.is_file():
+                        cmd_runner.send_files(str(ssl_file),
+                                              str(SCYLLA_SSL_CONF_DIR / ssl_file.name),
+                                              verbose=True)
+                stress_cmd = f'{stress_cmd.strip()} -tls -tls-ca-cert-file {SCYLLA_SSL_CONF_DIR}/ca.pem'
+
+                if self.params.get("client_encrypt_mtls"):
+                    stress_cmd = (f'{stress_cmd.strip()} -tls-client-key-file {SCYLLA_SSL_CONF_DIR}/test.key '
+                                  f'-tls-client-cert-file {SCYLLA_SSL_CONF_DIR}/test.crt')
+                if self.params.get("validate_hostname"):
```

Comment:
there's not such SCT configuration option, and if we do introduce it, we might want a better name for it.

Reply:
we can leave the whole code under comment, until someone would be addressing it.

Reply:
addressed - hostname validation control is to be implemented after it's fixed in s-b